.codecov.yml

```diff
@@ -1,9 +1,12 @@
 ---
 coverage:
   range: 70..100
-  round: down
+  round: nearest
   precision: 1
 
+ignore:
+  - "src/pudl/validate.py"
+
```

Comment:
I've tried many times to integrate collecting coverage into `make nuke` (which uses Dagster to run the ETL) and it never seems to come out right. Probably for reasons related to why Jan has had such a hard time collecting coverage in his CI revamp. So I've kind of given up on it.

I think the general plan is to move the data validation into the ETL process itself, rather than having it be a post-processing / testing step, which would bring this (or whatever it evolves into) into the fold of the main coverage collection system. Also this file has the *most* uncovered lines of any file and I hate seeing it, knowing I can never fix it.  I realize this is not particularly rational.  I just want us to be an A- student, not a B+ student okay man?